src/views/Profile.vue

```diff
@@ -1,90 +1,104 @@
 <template>
-    <v-card
-    class="mx-auto pa-2 mt-5 orange lighten-4"
-     max-width="95%"
-  >
+  <v-card class="mx-auto pa-2 mt-5 orange lighten-4" max-width="95%">
     <v-list-item three-line>
       <v-list-item-content>
         <v-list-item-title class="title mt-6  font-weight-bold">
-          {{ this.username }}'s Profile 
+          {{ this.username }}'s Profile
         </v-list-item-title>
-        <v-list-item-title >{{this.houseName}}</v-list-item-title>
-
-   
+        <v-list-item-title>{{ this.houseName }}</v-list-item-title>
       </v-list-item-content>
 
       <v-snackbar v-model="snackbar" timeout="2000">
-      Copied to Clipboard!
+        Copied to Clipboard!
       </v-snackbar>
 
-      
+      <v-avatar size="83" class="mb-3">
+        <Avatar
+          v-if="!this.$store.state.user.photo_url"
```

Comment:
Would it be best to place this store data in data or computed? Email and username seemed to be getting fetched so perhaps we could streamline the process to keep the code up here short/consistent?

Reply:
Since this isn't an issue with the emaildevs button, let's take it up elsewhere?